docs/walkthrough/updates.rst

```diff
@@ -28,17 +23,17 @@ with one additional line:
 .. code-block:: python
     :linenos:
     :caption: ephemeral.py
-    :emphasize-lines: 23
+    :emphasize-lines: 24
 
     @kopf.on.create('zalando.org', 'v1', 'ephemeralvolumeclaims')
-    def create_fn(spec, meta, namespace, logger, **kwargs):
+    def create_fn(body, spec, meta, namespace, logger, **kwargs):
 
         name = meta.get('name')
         size = spec.get('size')
         if not size:
             raise kopf.HandlerFatalError(f"Size must be set. Got {size!r}.")
 
-        path = os.path.join(os.path.dirname(__file__), 'pvc-tpl.yaml')
+        path = os.path.join(os.path.dirname(__file__), 'pvc.yaml')
         tmpl = open(path, 'rt').read()
         text = tmpl.format(size=size, name=name)
         data = yaml.load(text)
```

Comment:
`yaml.load(text)` --> `yaml.load(text, Loader=yaml.FullLoader)` because https://github.com/yaml/pyyaml/wiki/PyYAML-yaml.load(input)-Deprecation

Reply:
Oh yes, I saw that warning already few times. Will fix that too.